vitals_messages.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+#include "./vitals_monitor.h"
+#include <map>
+#include <string>
+
+// Combined message map with language as the outer index
+static const std::map<Language, std::map<VitalType,

```

Comment:
the strings aren't automatically tested. Also, it's hard to test all the situations in all the languages manually each time. Of course, the "understand-ability" of the translation can only be confirmed manually by native speakers.
But other things can go wrong, like missing a translation in a particular language, etc. Can you think of any asserts to safeguard against that?